source/statements.tex

```diff
@@ -672,8 +672,8 @@
 class type \tcode{C}, the \grammarterm{unqualified-id}{s}
 \tcode{begin} and \tcode{end} are looked up in the scope of \tcode{C}
 as if by class member access lookup~(\ref{basic.lookup.classref}), and if either
-(or both) finds at least one declaration, \grammarterm{begin-expr} and
-\grammarterm{end-expr} are \tcode{__range.begin()} and \tcode{__range.end()},
+(or both) finds at least one declaration, \textit{begin-expr} and
```

Comment:
Every other occurrence of *meow-expr* in this very section uses `\textit`, which is why I used them here.